docs/api/index.ipynb

```diff
@@ -0,0 +1,1055 @@
+       "iso_code date                                               \n",
+       "AFG      2020-02-24                                   NaN   \n",
+       "         2020-02-25                                   NaN   \n",
+       "         2020-02-26                                   NaN   \n",
+       "         2020-02-27                                   NaN   \n",
+       "         2020-02-28                                   NaN   \n",
+       "\n",
+       "                     excess_mortality_cumulative  excess_mortality  \\\n",
+       "iso_code date                                                        \n",
+       "AFG      2020-02-24                          NaN               NaN   \n",
+       "         2020-02-25                          NaN               NaN   \n",
+       "         2020-02-26                          NaN               NaN   \n",
+       "         2020-02-27                          NaN               NaN   \n",
+       "         2020-02-28                          NaN               NaN   \n",
+       "\n",
+       "                     excess_mortality_cumulative_per_million  \n",
+       "iso_code date                                                 \n",
+       "AFG      2020-02-24                                      NaN  \n",
+       "         2020-02-25                                      NaN  \n",
+       "         2020-02-26                                      NaN  \n",
+       "         2020-02-27                                      NaN  \n",
+       "         2020-02-28                                      NaN  \n",
+       "\n",
+       "[5 rows x 65 columns]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "catalog.find('covid').load().head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da49d07f-9f35-4d8b-9ec5-1f7b877ecc93",
+   "metadata": {},
+   "source": [
+    "## Metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04c2330b-53e5-4877-85c7-14a8a7cd0beb",
+   "metadata": {},
+   "source": [
+    "Each data frame loaded using our API also comes with accompanying metadata, which you can access by inspecting the `.metadata` object of the data frame:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "78dcbb32-f369-4463-97ae-c39950a58dec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = catalog.find('energy_mix').iloc[0].load()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "26aad110-a7c6-49d3-8048-6f48bcd78cbe",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "short_name\n",
+      "title\n",
+      "description\n",
+      "dataset\n",
+      "primary_key\n"
+     ]
+    }
+   ],
+   "source": [
+    "metadata = df.metadata\n",
+    "for f in metadata.__dataclass_fields__:\n",
```

Comment:
I didn't even know `__dataclass_fields__` exists :). I think it would be more useful to show metadata of some variable (e.g. `df.energy_consumption.metadata.title`) or show both how to access variable metadata and table metadata.

Reply:
I originally wanted to show the entire metadata object, but they're huge and hard to grok at a glance. I did your second suggestion instead, and showed access to metadata at various levels.